### PR TITLE
259 dg planned enhancements

### PR DESCRIPTION
**Pull request by juliusgambe**
Re Pull Request

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -105,7 +105,8 @@ The `UI` component,
 
 ### Logic component
 
-**API** : [`Logic.java`](https://github.com/AY2324S1-CS2103T-T13-3/tp/tree/master/src/main/java/seedu/address/logic/Logic.java)
+**
+API** : [`Logic.java`](https://github.com/AY2324S1-CS2103T-T13-3/tp/tree/master/src/main/java/seedu/address/logic/Logic.java)
```
Comment (zhonghan721) [inline docs/DeveloperGuide.md]: Should this be on the same line?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -146,7 +147,8 @@ How the parsing works:
 
 ### Model component
 
-**API** : [`Model.java`](https://github.com/AY2324S1-CS2103T-T13-3/tp/tree/master/src/main/java/seedu/address/model/Model.java)
+**
+API** : [`Model.java`](https://github.com/AY2324S1-CS2103T-T13-3/tp/tree/master/src/main/java/seedu/address/model/Model.java)
```
Comment (zhonghan721) [inline docs/DeveloperGuide.md]: Same as above

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -195,7 +198,8 @@ The `Customer` model,
 
 ### Storage component
 
-**API** : [`Storage.java`](https://github.com/AY2324S1-CS2103T-T13-3/tp/tree/master/src/main/java/seedu/address/storage/Storage.java)
+**
+API** : [`Storage.java`](https://github.com/AY2324S1-CS2103T-T13-3/tp/tree/master/src/main/java/seedu/address/storage/Storage.java)
```
Comment (zhonghan721) [inline docs/DeveloperGuide.md]: Same as above

**Review by zhonghan721 — APPROVED** [review]
Some comments, otherwise LGTM!

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (juliusgambe) [reply docs/DeveloperGuide.md]: Thanks for the catch. Due to the line wrap at 120, I have changed it to:
```
**API**:
[`Logic.java`](https://github.com/AY2324S1-CS2103T-T133/tp/tree/master/src/main/java/seedu/address/logic/Logic.java)
```

etc.

**Pull request merged**